common/src/main/java/com/wynntils/models/abilities/type/OphanimOrb.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.abilities.type;
+
+public class OphanimOrb {
+    private final HealthState healthState;
+
+    public OphanimOrb(HealthState healthState) {
+        this.healthState = healthState;
+    }
+
+    public String getString() {
+        return healthState.getColorCode() + "⏺";
+    }
+
+    public enum HealthState {
+        HEALTHY("§b"),
+        DAMAGED("§c"),
+        DYING("§e"),
+        DEAD("§7");
+
+        private final String colorCode;
```

Comment:
It would be better to store the underlying ChatFormatting here, and convert it into string, when needed.